web/wp-content/themes/dctx-v2/single.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-efuzga.php, but found phpcs-scan-eFUZGA.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).